### DISCUSSION TITLE
Persistence of aliases and sub-domains?

### DISCUSSION THREAD

**Discussion by vjayer** [simple-login/app#1747] [Q&A] [answered]
If for any reason we cancel or do not renew our premium subscription in time, can we still regain access to our previously generated aliases and subdomains when we sign up again?

**Comment by fm** [comment]
Your existing aliases will continue to work when you downgrade to free, you just won't be able to create new ones.

**Reply by vjayer** [reply]
Thanks for your reply. Does that also apply to custom subdomains created in premium too? I ask since those and corresponding aliases are not available in free tier.

**Reply by vjayer** [reply, marked as answer]
Found out the answer to question about custom subdomains is yes, those will persist too.

**Reply by THEOCKID** [reply]
HEY! quick tag-along question:  OK so I am/was totally new to ProtonMail/Simple Login.   Right off the bat created 5 subdomains (and Im a paid subscriber).  But now too late I realize I don't want 2-3 of them and want to create 2-3 new subdomains.

What if I cancel my current subscription (with Subdomains A B C D E) and then start a new subscription and generated my desired subdomains F G H?  I'll still have access to A B C D and E, and then get the 3 new subdomains I want.

Sound do-able?

**Reply by nguyenkims** [reply]
@THEOCKID the quota can't be reset this way unfortunately. You can however reach out to the support team at https://app.simplelogin.io/dashboard/support and they can find a solution.

**Reply by vjayer** [reply]
There's a related feature request that I commented on about this here: https://github.com/simple-login/app/discussions/1031#discussioncomment-5983920 (Please add option to pay for extra subdomains #1031)

It's good to know that support might be able to manual reset the quota / remove subdomains if that's what you mean. Otherwise, as another complimentary feature request--I know, you've got lots already--is to provide a way delete them and return back to the unallocated quota

**Comment by THEOCKID** [comment]
I just opened another simple login account.  Screw it.  30 a year is peanuts.  Now I'll think lore carefully before creating subdomains. 

Jun 14, 2023 9:29:21 AM Son Nguyen Kim ***@***.***>:
